From 0c381f08f361d232c752e74fab9c05ffc2cdec43 Mon Sep 17 00:00:00 2001
From: "LAPTOP-E3NS1O95\\\\User" <madgalene@insidersecurity.co>
Date: Tue, 7 Sep 2021 19:24:59 +0800
Subject: [PATCH 1/8] update

---
 gitPatch.py | 9 ---------
 1 file changed, 9 deletions(-)
 delete mode 100644 gitPatch.py

diff --git a/gitPatch.py b/gitPatch.py
deleted file mode 100644
index 05444a9..0000000
--- a/gitPatch.py
+++ /dev/null
@@ -1,9 +0,0 @@
-import git
-import os
-
-# git clone repository
-#git.Git("C:\\Users\\User\\git_projects").clone("https://github.com/921am/ccTruthBasket.git")
-
-# git patch
-# os.system("git checkout main")
-os.system("git format-patch main")
\ No newline at end of file
-- 
2.33.0.windows.2

